extensions/control-plane/transfer/transfer-data-plane/src/main/java/org/eclipse/edc/connector/transfer/dataplane/flow/ConsumerPullTransferDataFlowController.java

```diff
@@ -35,6 +35,7 @@
 import static org.eclipse.edc.spi.response.StatusResult.failure;
 import static org.eclipse.edc.spi.types.domain.transfer.FlowType.PULL;
 
+@Deprecated
```

Comment:
nit: i would put a `"since"` annotation here, and add javadoc with the `@deprecated` tag